### PR TITLE
Colin not required as assignee on client dev tasks

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

**Review by ripienaar — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/client_enhancement.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Client Enhancement
 about: Request a new feature to be added to NATS clients
 labels: enhancement, client
-assignees: ripienaar, wallyqs, scottf, aricart, ColinSullivan1, kozlovic, derekcollison, matthiashanel, Jarema
```
Comment (ColinSullivan1) [inline .github/ISSUE_TEMPLATE/client_enhancement.md:5]: I'm fine being removed from the .NET client feature requests. But in terms of high level / overall feature requests it's a good idea to keep the NATS product manager involved.  :)

**Review by ColinSullivan1 — CHANGES_REQUESTED** [review]
Product management still needs to be involved at a high level

File: .github/ISSUE_TEMPLATE/client_enhancement.md (hunk 1 above)
Comment (scottf) [reply .github/ISSUE_TEMPLATE/client_enhancement.md]: fixed

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.

**Pull request merged**